src/value_generators/src/value_generators.cpp

```diff
@@ -33,5 +39,444 @@ std::unique_ptr<DocumentGenerator> makeDoc(const YAML::Node node, genny::Default
         return std::unique_ptr<DocumentGenerator>{new TemplateDocument(node, rng)};
 };
 
+namespace {
+
+const auto parserMap = std::unordered_map<std::string, Expression::Parser>{
+    {"^FastRandomString", FastRandomStringExpression::parse},
+    {"^RandomInt", RandomIntExpression::parse},
+    {"^RandomString", RandomStringExpression::parse}};
+
+}  // namespace
+
+template <class... Ts>
+struct overloaded : Ts... {
+    using Ts::operator()...;
+};
+
+template <class... Ts>
+overloaded(Ts...)->overloaded<Ts...>;
+
+int64_t Value::getInt64(std::string_view name) {
+    int64_t ret;
+
+    std::visit(
```

Comment:
[note] Using `std::visit()` requires macOS 10.14 (Mojave) which may be problematic if people haven't upgraded already. (I hadn't.)

https://stackoverflow.com/questions/52310835/xcode-10-call-to-unavailable-function-stdvisit